components/Graphs/MostUsedLanguagesGraph/most-used-languages-graph.tsx

```diff
@@ -112,23 +112,15 @@ export const MostUsedLanguagesGraph = ({
                   key={item.name}
                   className={`flex justify-between pt-4 pb-4 ${
                     index === lastItem ? "" : "border-b-1 border-slate-100"
-                  }`}
+                  } ${language === item.name ? "font-semibold" : ""}`}
```

Comment:
The changes in this file are back to what I originally had in #2158 where the `font-semibold` is inherited by the child elements.